pkg/operator/ceph/object/admin.go

```diff
@@ -75,6 +76,17 @@ func RunAdminCommandNoMultisite(c *Context, args ...string) (string, error) {
 	return output, nil
 }
 
+func ExtractJson(output string) (string, error) {
```

Comment:
Have you thought about putting this inside the Executor directly?

Reply:
Can we call `ExtractJson()` from `RunAdminCommandNoMultisite()`? That seems like the right place instead of calling it everywhere else.

Reply:
Year, it's nice idea. The new PR is far simpler than before.

I introduced a flag in `runAdminCommand()` to decide whether calling `extractJSON()` inside them. It's because some callers checking output before parsing JSON. In this case, that calllers directly call `extractJSON()` after checking output.